htcanalyze/job_details.py

```diff
@@ -0,0 +1,71 @@
+"""Save HTCondor job execution details."""
+
+
+class JobDetails:
+    """
+    Class to store and manage the different job details.
+
+    Mostly the complexity lies in creating
+    colored output depending on the states
+    """
+
+    state_colors = {
+        'abnormal': 'red',
+        'aborted': 'red',
+        'ewr': 'red',  # error while reading
```

Comment:
why not make this `error_while_reading`?

Reply:
Will do